core-bundle/src/Image/ImageHelper.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Image;
+
+use Contao\CoreBundle\Asset\ContaoContext;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\FilesModel;
+use Contao\Frontend;
+use Contao\Image\PictureConfiguration;
+use Contao\Validator;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class ImageHelper
+{
+    public const PICTURE_IMAGE = 'img';
+    public const PICTURE_SOURCES = 'sources';
+    public const PICTURE_ALT = 'alt';
+
+    /** @var PictureFactoryInterface */
+    private $pictureFactory;
+
+    /** @var string */
+    private $rootDir;
+
+    /** @var string */
+    private $staticUrl;
+
+    /** @var string */
+    private $locale;
+
+    /** @var ContaoFramework */
+    private $framework;
+
+    public function __construct(PictureFactoryInterface $pictureFactory, string $rootDir, ContaoContext $fileContext, RequestStack $requestStack, ContaoFramework $framework)
+    {
+        $this->pictureFactory = $pictureFactory;
+        $this->rootDir = $rootDir;
+        $this->staticUrl = $fileContext->getStaticUrl();
+        $this->framework = $framework;
+
+        $request = $requestStack->getCurrentRequest();
+        $this->locale = null !== $request ? $request->getLocale() : '';
```

Comment:
should this use the system default local (`en` usually configured in the framework) instead of an empty value?